src/text/helper.ts

```diff
@@ -77,6 +77,18 @@ const getFontWeight = (weight: TextWeight) => {
 // =============================================================================
 // TEXT STYLE
 // =============================================================================
+const getMaxLinesLineStyle = (maxLines: number | undefined = undefined) => {
+    if (maxLines > 0) {
+        return css`
+            display: -webkit-box !important;
```

Comment:
is this `!important` required?

Reply:
yes, need, otherwise display: 'block' overwrite it based on the testing.

Reply:
removed and code submitted